src/jimgw/core/single_event/likelihood.py

```
@@ -466,7 +466,7 @@ def __init__(
         n_bins: int = 100,
         popsize: int = 100,
         n_steps: int = 2000,
-        ref_params: dict = {},
+        reference_parameters: dict = {},
```

Ruff (0.14.13)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)